src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -112,6 +118,20 @@ private User getUser(String username) {
                 getExceptionMessageSource("exception.user.not-found")));
     }
 
+    private void checkReadOnlyFieldsUpdate(User user, UserDto userDto) throws IllegalOperationException {
```

Comment:
In my oppinion it would be better to check all read-only fields and throw an exception with information about them all, than doing the same but only for one field at the moment.